src/Formatter/CallableFormatter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace League\BooBoo\Formatter;
+
+use Exception;
```

Comment:
I'm torn on this, but I tend to agree. Not worth blocking a PR from a volunteer for, though.